doc/acting/discontinued/teb/motion_planner.py

```
@@ -0,0 +1,1028 @@
+    TARGET_DISTANCE_TO_STOP,
+    convert_to_ms,
+    spawn_car,
+)
+
+
+sys.path.append(os.path.abspath(sys.path[0] + "/../../planning/src/behavior_agent"))
+from behaviors import behavior_speed as bs  # type: ignore # noqa: E402
+
+from teb_planner_pa_msgs.srv import Plan, PlanRequest, PlanResponse
+from mapping.msg import Map as MapMsg
+from mapping_common.entity import Entity
```

Ruff (0.8.2)

`mapping_common.entity.Entity` imported but unused

Remove unused import: `mapping_common.entity.Entity`

(F401)